lib/rocket_chat/messages/group.rb

```diff
@@ -98,7 +98,7 @@ def online(room_id: nil, name: nil)
       # * [String] topic A room's topic
       # * [Strong] type c (channel) or p (private group)
       def self.settable_attributes
-        %i[description purpose read_only topic type]
+        %i[custom_fields description purpose read_only topic type]
```

Comment:
Same as above, although groups only have the `custom_fields` and `announcement` setters (no `setDefault`)

```suggestion
        %i[announcement custom_fields description purpose read_only topic type]
```

(and a similar comment change too! 😄 )